data/json/mapgen/godco/nested.json

```diff
@@ -0,0 +1,367 @@
+[
+    {
```

Comment:
Also needs linting.  Please ping me when most errors have stopped popping up and I'll come review this.

Reply:
Looks like you were linting stuff while I was reviewing :)

Reply:
I was. I'll let you know when most of the errors have stopped.